CONTRIBUTING.md

```diff
@@ -89,6 +92,60 @@ Use the test scripts provided in the `ci/` directory to run tests for each compo
 ./ci/test_cub.sh -cxx g++ -std 14 -arch "70;75;80-virtual"
 ```
 
+### Using CMake Presets
+
+[CMake Presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html) are a set of configurations defined in a JSON file that specify project-wide build details for CMake. They provide a standardized and sharable way to configure, build, and test projects across different platforms and development environments. Presets are available from CMake version 3.19 and later.
+
+There are three kinds of Presets:
+
+- Configure Presets: specify options for the `cmake` command.
+
+- Builds Presets: specify options for the `cmake --build` command.
+
+- Test Presets: specify options for the `ctest` command.
```

Comment:
Bullet lists have two grammatically correct options.  You can either 

1. make everything but the bullets a complete sentence, and treat the bullets as sentence fragments, or
2. treat the bullets as part of the text, and make everything a complete sentence.

In the suggestion below, I've chosen Option (2). 

```suggestion
There are three kinds of Presets.

- Configure Presets: specify options for the `cmake` command

- Build Presets: specify options for the `cmake --build` command

- Test Presets: specify options for the `ctest` command
```